wagtail/images/views/images.py

```diff
@@ -80,7 +80,13 @@ def get_paginate_by(self, queryset):
         return getattr(settings, "WAGTAILIMAGES_INDEX_PAGE_SIZE", 30)
 
     def get_valid_orderings(self):
-        return self.ORDERING_OPTIONS
+        orderings = self.ORDERING_OPTIONS.copy()
+        if self.is_searching:
+            # Ordering by usage count not currently available when searching,
+            # due to https://github.com/wagtail/django-modelsearch/issues/12
```

Comment:
Should probably update this to https://github.com/wagtail/django-modelsearch/issues/51 instead